lib/puppet/file_serving/metadata.rb

```diff
@@ -50,7 +56,7 @@ class WindowsStat < MetaStat
       require 'puppet/util/windows/security'
     end
 
-    def initialize(stat, path, source_permissions = nil)
```

Comment:
@joshcooper one windows spec test is failing because it was depending on `source_permissions` being `nil` by default here. Could that be a problem elsewhere, or do we expect to always pass something in? (If `nil` or `ignore` isn't explicitly passed in here, `@source_permissions_ignore` will not be set. Shouldn't it be true by default now?)